src/mtcp/mtcp_restart.c

```diff
@@ -207,7 +207,7 @@ MTCP_PRINTF("Attach for debugging.");
   while (argc > 0) {
     // Flags for standalone debugging
     if (argc == 1) {
-      MTCP_PRINTF("Considering '%s' as a ckpt image.\n", argv[0]);
+      mtcp_printf("Considering '%s' as a ckpt image.\n", argv[0]);
```

Comment:
Could you also add a comment here that states the fact that this condition is true only for readmtcp and that's why it's okay to use mtcp_printf here. Alternatively, could we just remove this line from here and move it in `mtcp_simulateread`?